articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+ 
+## Network
+
+UKCloud manage the physical firewalls that face public and secure networks.
+
+Customers can deploy mutliple gateway devices and configure the following using either the Azure Stack Portal GUI or API:
+
+- Virtual networks – UKCloud for Microsoft Azure allows customers to configure virtual private clouds, and allows them to create and manage their private address space within Azure. This can be configured via the UI, API and other tooling 
+- Load balancers (IPv4 only) – customers can deploy basic load balancers o VPN gateway – the UKCloud for Microsoft Azure VPN gateway can be used to create site-to-site VPNs to remote infrastructure 
+- Routing – basic custom defined routing can be configured on virtual networks 
+- SSL VPN?
+
+**Internet-facing solutions** have 8 usable public IP addresses. You can ask for additional public IP addresses via a Service Request. We reserve the right to decline the request if you have spare capacity in your existing deployment.
+
+## Advanced features 
+- Key Vault. UKCloud for Microsoft Azure provides cryptographic secret storage for applications and services. These services can be consumed by both portal and API.  
+https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-kv-intro
+
+## Protection options
+
+Customers are responsible for designing their own disaster recovery and backup solutions as part of their adoption of UKCloud for Microsoft Azure.
+
+## Service availability
+
+The availability of your virtual machines in UKCloud for Microsoft Azure follows the same principles of the Microsoft Azure Public Cloud, where customers achieve a higher service level availability based on how they architect the solution. 
+
+The service levels are as follows:
+
+- 99.90% (a single VM deployed as a single instance deployed inside and availability zone)
+- 99.95% (a single VM with two instances deployed inside an availability zone)
+
+> [!TIP]
+> To qualify for the higher service level, customers are advised to use on platform tools like availability sets that ensures your VMs are provisioned to different hardware in the region.
+
+## Service background
+
+- VMs are fully provisioned.
+
+- The system is configured to automatically balance resources, so your VMs may be moved between physical hosts.
+
+- We actively capacity-manage the cloud platform to ensure you have access to the resources you request.
+
+- You can specify the site where you would like to have your service provisioned (from sites where UKCloud for Microsoft Azure is available). 
+
+- The UKCloud for Microsoft Azure service is updated regularly with the latest code release from Microsoft - typically, this is once a month and notified via the [UKCloud service status page](https://status.ukcloud.com/) and the [UKCloud portal](https://portal.skyscapecloud.com/login). UKCloud aim to be within two updates from Microsoft, however, we will control the release of the patches into the live service.
```

Comment:
UKCloud Portal (uppercase P)

URL for Portal should be https://portal.ukcloud.com